app/views/analytics/_customer_analytics.haml

```diff
@@ -1,38 +1,17 @@
-- if feature_enabled?(:customer_universal_analytics)
-  :plain
-    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','//www.google-analytics.com/analytics.js','ga_customer');
+:plain
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','//www.google-analytics.com/analytics.js','ga_customer');
 
-    var community_ga_key = "#{@current_community.google_analytics_key}"
-    var community_cookie_domain = "#{PublicSuffix.parse(request.host).domain}"
-    ga_customer('create', community_ga_key, 'auto', {'legacyCookieDomain': community_cookie_domain,   'allowLinker': true});
-    ga_customer('send', 'pageview');
-    var secondary_analytics_in_use = true;
+  var community_ga_key = "#{@current_community.google_analytics_key}"
+  var community_cookie_domain = "#{PublicSuffix.parse(request.host).domain}"
+  ga_customer('create', community_ga_key, 'auto', {'legacyCookieDomain': community_cookie_domain,   'allowLinker': true});
+  ga_customer('send', 'pageview');
+  var secondary_analytics_in_use = true;
```

Comment:
Namespacing `ga_customer` is scary, because it's created (and possibly referenced) by asynchronously loaded analytics scripts. I'm not sure of a safe way to do that. Cleaned the other things from global namespace, though